server/cmd_auth.go

```diff
@@ -18,15 +18,25 @@ type Select struct {
 	commands.Select
 }
 
-func (cmd *Select) Handle(conn Conn) error {
+func (cmd *Select) Handle(conn Conn) (err error) {
```

Comment:
One last nit: can you revert the change that switches to named return values?

See this for details: https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

Reply:
oh yes :man_facepalming: sure